From 1f93034050b38029447a201bc5d4305fde70b1b7 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Sun, 12 Oct 2025 18:26:40 +0200
Subject: [PATCH] disable werror on linux port

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 port/platform/linux/linux.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/port/platform/linux/linux.cmake b/port/platform/linux/linux.cmake
index e5233573..ebe3e11a 100644
--- a/port/platform/linux/linux.cmake
+++ b/port/platform/linux/linux.cmake
@@ -24,8 +24,8 @@ message("UBXLIB_FEATURES will be \"${UBXLIB_FEATURES}\"")
 set(CMAKE_BUILD_TYPE Debug)
 # The Posix implementations require these libraries
 set(UBXLIB_REQUIRED_LINK_LIBS -lm -lssl -lpthread -lrt -lgpiod)
-# Warnings are errors
-add_compile_options(-Wall -Werror -Wno-format-truncation -Wno-stringop-truncation -funsigned-char)
+# Warnings are warnings
+add_compile_options(-Wall -Wno-format-truncation -Wno-stringop-truncation -funsigned-char)
 
 # Add any #defines specified by the environment variable U_FLAGS
 if (DEFINED ENV{U_FLAGS})
-- 
2.51.0

