From e342bdbbf1a355dab9fbd74e4811dd0da6644513 Mon Sep 17 00:00:00 2001
From: Jon Nettleton <jon@solid-run.com>
Date: Sun, 6 Feb 2022 10:42:24 +0100
Subject: [PATCH 05/11] ASoC: SOF: Add missing include for arm_smccc_smc

Signed-off-by: Jon Nettleton <jon@solid-run.com>
---
 sound/soc/sof/imx/imx8ulp.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/sof/imx/imx8ulp.c b/sound/soc/sof/imx/imx8ulp.c
index f020c5c138ad..db899d3f8038 100644
--- a/sound/soc/sof/imx/imx8ulp.c
+++ b/sound/soc/sof/imx/imx8ulp.c
@@ -6,6 +6,7 @@
 //
 // Hardware interface for audio DSP on i.MX8ULP
 
+#include <linux/arm-smccc.h>
 #include <linux/clk.h>
 #include <linux/firmware.h>
 #include <linux/of_platform.h>
-- 
2.35.1

