From e2c37ec9d7f6d21a6b49ee5042996e04242bf6cd Mon Sep 17 00:00:00 2001
From: TalPilo <tal.pilo@solid-run.com>
Date: Thu, 21 Nov 2019 16:28:06 +0200
Subject: [PATCH] Add distro bootcmd support

Signed-off-by: TalPilo <tal.pilo@solid-run.com>
---
 include/configs/imx8mm_solidrun.h | 37 ++++++++++++++++++-------------
 1 file changed, 22 insertions(+), 15 deletions(-)

diff --git a/include/configs/imx8mm_solidrun.h b/include/configs/imx8mm_solidrun.h
index f9aec5ec11..237b82089e 100644
--- a/include/configs/imx8mm_solidrun.h
+++ b/include/configs/imx8mm_solidrun.h
@@ -42,6 +42,7 @@
 #define CONFIG_SYS_ICACHE_OFF
 #define CONFIG_SYS_DCACHE_OFF
 
+
 #define CONFIG_MALLOC_F_ADDR		0x912000 /* malloc f used before GD_FLG_FULL_MALLOC_INIT set */
 
 #define CONFIG_SPL_ABORT_ON_RAW_IMAGE /* For RAW image gives a error info not panic */
@@ -155,10 +156,18 @@
 #define CONFIG_EXTRA_ENV_SETTINGS		\
 	CONFIG_MFG_ENV_SETTINGS \
 	JAILHOUSE_ENV \
+	BOOTENV	\
+	"pxefile_addr_r=0x40480000\0" \
 	"script=boot.scr\0" \
 	"image=Image\0" \
 	"console=ttymxc1,115200 earlycon=ec_imx6q,0x30890000,115200\0" \
-	"fdt_addr=0x43000000\0"			\
+	"fdtfile=undefined\0" \
+	"ramdisk_addr_r=0x50000000\0"	\
+        "fdt_addr_r=0x43000000\0" \
+        "fdt_addr=0x43000000\0" \
+        "kernel_addr_r=0x40480000\0" \
+        "pxefile_addr_r=0x40480000\0" \
+        "scriptaddr=0x40480000\0" \
 	"fdt_high=0xffffffffffffffff\0"		\
 	"boot_fdt=try\0" \
 	"fdt_file=" CONFIG_DEFAULT_FDT_FILE "\0" \
@@ -204,21 +213,10 @@
 			"fi; " \
 		"else " \
 			"booti; " \
-		"fi;\0"
-
-#define CONFIG_BOOTCOMMAND \
-	   "mmc dev ${mmcdev}; if mmc rescan; then " \
-		   "if run loadbootscript; then " \
-			   "run bootscript; " \
-		   "else " \
-			   "if run loadimage; then " \
-				   "run mmcboot; " \
-			   "else run netboot; " \
-			   "fi; " \
-		   "fi; " \
-	   "else booti ${loadaddr} - ${fdt_addr}; fi"
+		"fi;\0"	
 #endif
 
+
 /* Link Definitions */
 #define CONFIG_LOADADDR			0x40480000
 
@@ -244,7 +242,7 @@
 #elif defined(CONFIG_ENV_IS_IN_NAND)
 #define CONFIG_ENV_OFFSET       (60 << 20)
 #endif
-#define CONFIG_ENV_SIZE			0x1000
+#define CONFIG_ENV_SIZE			0x2000
 #define CONFIG_SYS_MMC_ENV_DEV		0   /* USDHC2 */
 #define CONFIG_MMCROOT			"/dev/mmcblk1p2"  /* USDHC2 */
 
@@ -386,6 +384,15 @@
 
 #define CONFIG_OF_SYSTEM_SETUP
 
+#define BOOT_TARGET_DEVICES(func) \
+        func(MMC, mmc, 1) \
+        func(MMC, mmc, 0) \
+        func(USB, usb, 0) \
+        func(PXE, pxe, na) \
+        func(DHCP, dhcp, na)
+
+#include <config_distro_bootcmd.h>
+
 #if defined(CONFIG_ANDROID_SUPPORT)
 #include "imx8mm_evk_android.h"
 #endif
-- 
2.17.1

