From d5371e2747500a22cac83f959701a6cb01e99436 Mon Sep 17 00:00:00 2001
From: yazan shhady <yazan.shhady@solid-run.com>
Date: Tue, 21 Dec 2021 22:09:04 +0200
Subject: [PATCH] Replace default evk.dtb with imx8mp-solidrun.dtb

---
 iMX8M/soc.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/iMX8M/soc.mak b/iMX8M/soc.mak
index 4bfdc37..b3e9990 100644
--- a/iMX8M/soc.mak
+++ b/iMX8M/soc.mak
@@ -139,7 +139,7 @@ u-boot-atf-tee.bin: u-boot.bin bl31.bin tee.bin
 clean:
 	@rm -f $(MKIMG) u-boot-atf.bin u-boot-atf-tee.bin u-boot-spl-ddr.bin u-boot.itb u-boot.its u-boot-ddr3l.itb u-boot-ddr3l.its u-boot-spl-ddr3l.bin u-boot-ddr4.itb u-boot-ddr4.its u-boot-spl-ddr4.bin u-boot-ddr4-evk.itb u-boot-ivt.itb u-boot-ddr4-evk.its $(OUTIMG)
 
-dtb = evk.dtb
+dtb = imx8mp-solidrun.dtb
 $(dtb):
 	./$(DTB_PREPROC) $(PLAT)-evk.dtb $(dtb) $(dtbs)
 
-- 
2.25.1

